docs/source/get_involved/code_of_conduct.rst

```diff
@@ -133,13 +165,21 @@ project event or platform include:
 -  nonconsensual or unwelcome physical contact
 -  sustained disruption of talks, events or communications
 -  insults or put downs
--  sexist, racist, homophobic, transphobic, ableist, or exclusionary
-   jokes
+-  sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
 -  excessive swearing
 -  incitement to violence, suicide, or self-harm
 -  continuing to initiate interaction (including photography or
    recording) with someone after being asked to stop
 -  publication of private communication without consent
+-  decision making on resource allocation which has the effect of excluding people
+   on the basis of protected characteristics, or other forms of systemic discrimination
+-  decision making on resource allocation that confers advantages to personal networks,
+   and excludes others, e.g., preventing advertising of opportunities, not applying best practice in recruitment
+-  reducing transparency of operations of sktime, e.g., by obfuscation of the paper trail
+-  invoking the code of conduct dishonestly, or as a form of bullying,
+   especially from a position of power
+
+Some more principled guidance is given in the appendix "Guidance in applying the CoC".
```

Comment:
can you kindly remove that part, this is in my opinion a very controversial section. What is the definition of "resource"? In my opinion we have above clarified already enough the the community  values and what kind of behavior we appreciate and do not. Why would that even require to be repeated here for "resource allocation"? Its cleat that those values are applied at any point in time and to any decision making process.

Reply:
> can you kindly remove that part, this is in my opinion a very controversial section.

This is the key part of the PR!

> Its cleat that those values are applied at any point in time and to any decision making process.

Am I correctly summarizing your point, that you think this is redundant since it follows from the community principles?

If you think it is redundant, I think it makes sense to list these: the section collects examples of unacceptable behaviour for the benefit of clarity.

The original purpose of this section is to have common misbehaviours called out as examples, as to preempt discussion on how the community principles are to be applied.

Or, do you think that the new examples are acceptable behaviour? Would appreciate some clarity here.